### DISCUSSION TITLE
[06.06.2024] Question: XIV Auras are not compatbile with game

### DISCUSSION THREAD

**Discussion by KonanHarana** [xivdev/Penumbra#423] [Q&A] [answered]
Hello Community, 

I am currently installing FFMods on my new PC. However I cant install XIVAuras via Dalamund because it states: 
"THis plugin is outdated and incompatible. Please wait for it to be updated by its author"

Is this project still running? Ive seen not that much activity here so I am wondering if I can use this feature in the future or if its currently not actively supported by anyone :) 

Many Thanks!

**Comment by Theo-Asterio** [comment, marked as answer]
I'm very unsure how you wound up in this repo exactly but I think there's a few things you're confused about.

In FFXIV mods are not the same thing as plugins, they're very very different things. Mods are more like visual things that are changed in the game, think of it like things that change how a piece of gear looks. Plugins are more akin to application like things in the game that add/tweak some functionality such as XIV Auras.

Penumbra is not a plugin loader, it loads mods such as TTMP2/TTMP/PMP. It has nothing to do with XIV Auras at all.
XIVLauncher/Dalamud is a framework that deals with loading plugins, such as XIV Auras.
Normally you'd ask in the XIVLauncher/Dalamud server but XIV Auras is a third party plugin so, instead you'd talk with the dev of said plugin or visit their github.

You are reporting this in a repo where the Dev of this plugin has nothing nor had anything ever to do with XIV Auras. I highly suggest finding the proper repo for XIV Auras via google or checking in the plugin installer if there's an Open Website button that takes you to where they accept feedback/questions.

**Reply by KonanHarana** [reply]
Oh lol, I posted in the wrong project :D 
Just ignore my post though!
